atlassian-terraform/terraform.tfvars

```diff
@@ -0,0 +1,93 @@
+
+# Sumo Logic Apps and Webhooks
+install_jira_cloud = "true"
+install_jira_on_prem = "true"
+install_bitbucket_cloud = "true"
+install_opsgenie = "true"
+# install_opsgenie should be true for the below option install_sumo_to_opsgenie_webhook to be true. https://help.sumologic.com/Manage/Connections-and-Integrations/Webhook-Connections/Webhook_Connection_for_Opsgenie
+install_sumo_to_opsgenie_webhook = "false" # This feature is in Beta. To participate contact your Sumo account executive. You can modify the file sumo_to_opsgenie_webhook.json.tmpl for customizing payload.
+install_atlassian_app = "true"
+install_sumo_to_jiracloud_webhook = "false" # This feature is in Beta. To participate contact your Sumo account executive. You can modify the file sumo_to_jiracloud_webhook.json.tmpl for customizing payload.
+install_sumo_to_jiraserver_webhook = "false" # This feature is in Beta. To participate contact your Sumo account executive. You can modify the file sumo_to_jiraserver_webhook.json.tmpl for customizing payload.
+
+# Jira Cloud
+jira_cloud_url = "https://<example>.atlassian.net"
+jira_cloud_user = ""
+jira_cloud_password = "" # Password or API Key, To generate API Key: https://confluence.atlassian.com/cloud/api-tokens-938839638.html
+jira_cloud_jql = "" #jql = "project = Sumo" See, https://support.atlassian.com/jira-software-cloud/docs/what-is-advanced-searching-in-jira-cloud/
+// See https://developer.atlassian.com/cloud/jira/platform/webhooks/ for supported events
+jira_cloud_events = ["jira:issue_created", "jira:issue_updated","jira:issue_deleted",
+"issue_property_set","issue_property_deleted","comment_created","comment_updated","comment_deleted",
+"worklog_created","worklog_updated","worklog_deleted",
+"attachment_created","attachment_deleted","issuelink_created","issuelink_deleted",
+"project_created","project_updated","project_deleted",
+"jira:version_released","jira:version_unreleased","jira:version_created","jira:version_moved","jira:version_updated","jira:version_deleted","jira:version_deleted",
+"user_created","user_updated","user_deleted",
+"option_voting_changed","option_watching_changed","option_watching_changed","option_unassigned_issues_changed","option_subtasks_changed","option_attachments_changed","option_issuelinks_changed","option_timetracking_changed",
+"sprint_created","sprint_deleted","sprint_updated","sprint_started","sprint_closed",
+"board_created","board_updated","board_deleted","board_configuration_changed",
+"jira_expression_evaluation_failed"]
+
+# Sumologic to Jira Cloud Webhook - This feature is in Beta. To participate contact your Sumo account executive.
+# https://developer.atlassian.com/cloud/jira/platform/rest/v2/#api-rest-api-2-issue-post
+jira_cloud_issuetype = "Bug"
+jira_cloud_priority = "3"
+jira_cloud_projectkey = "project"
+jira_cloud_auth = "" #Basic Authorization Header. See: https://help.sumologic.com/Beta/Webhook_Connection_for_Jira_Cloud#prerequisite
```

Comment:
Similarly, is this a required parameter, or an empty string is the default value?

Reply:
Added